proto/ociregistry.proto

```diff
@@ -51,6 +51,9 @@ message OCIActionResultKey {
 
   // Digest as a hex string.
   string hash_hex = 5;
+
```

Comment:
Possibly dumb question, but is OCIActionResultKey persisted anywhere? Does it need to be a proto message or can it be a struct?

Reply:
Ok, let's save that for a separate change.

Reply:
Not as a one liner, but you can manually feed all the struct members into a hasher.